Jenkinsfile

```diff
@@ -477,7 +479,70 @@ EOL
                 }
             }
         }
-	}
+        stage('Package helm charts'){
```

Comment:
Can we do it in parallel with some other stage? Docker build for example...

Reply:
Refactored. Moved stages to parallel stages